### ISSUE TITLE
Agent with cloudbeat installed and running is marked as unhealthy

### ISSUE THREAD

**Issue by jeniawhite** [elastic/cloudbeat#465]
**Describe the bug**
I have an installed agent with cloudbeat running and delivering findings as expected.
But when I open the fleet UI, I can see that the agent's status is unhealthy.

**Preconditions**
Environment state and/or platform specific configurations.

**To Reproduce**
Write the exact actions one should perform in order to reproduce the bug.
1.

**Expected behavior**

**Comment by jeniawhite** [comment]
There were two agents that are part of a kind cluster deployed on EC2 machines.
The allocated disk space (single partition) for the EC2 was 16GB.
I've noticed an infinite amount of errors regarding space both in the logs and when connecting remotely to the machines (ssh).
When successfully connected to the machine, I couldn't perform any actions due to being out of space.
`kubectl` and other operations did not work and I couldn't connect to the `elastic-agent` pod in order to run the status.
When investigating what took up all of the space, I've come to the conclusion that the docker daemon that runs the kind cluster takes up all of the 16GB.
I've attempted to see if I can prune any non-usable assets from the docker daemon like `images` and virtual drives and it seemed like everything is being utilized and that there is no unnecessary waste.
Right now I'm at a stage where I cannot connect (ssh) or ping any of the machines, yet they are still sending check-ins to the `fleet` while remaining to be unhealthy.
I've attempted to dynamically resize the disk of one of the machines, but due to not being able to connect (ssh) to it I cannot extend the file system even though the disk resized successfully via the EC2 console.
My advice is to allocate more than 16GB for an EC2 instance that runs a kind cluster.
